### PR TITLE
iOS starting point for Firebase mojo service

### PR DESCRIPTION
**Pull request by collinjackson**

### CODE REVIEW

File: third_party/firebase/Firebase.framework/Headers/FAuthData.h [hunk 1]
```diff
@@ -0,0 +1,50 @@
+//
```
Comment (abarth) [inline third_party/firebase/Firebase.framework/Headers/FAuthData.h]: Rather than checking this binary blob into git, can we grab it from cloud storage?

**Comment by Hixie** [comment]
How do you build the binary blob?

**Comment by collinjackson** [comment]
The binary blob is not open source. It came from https://github.com/firebase/firechat-ios/blob/master/Firebase.framework/Versions/A/Firebase

I'm going to update to a newer version anyway since this one is over a year old. I will try the cloud storage approach.

We'll probably move some or all of the Firebase stuff out of shell and into a separate repo once we have a good workflow for native extensions.

**Comment by Hixie** [comment]
@collinjackson Same deal with this one; feel free to reopen if this is still going to be checked in in this form at some point.

**Pull request closed without merging**